AssistEm/assistem-server
# user's profile picture (node and angular)

implemented profile picture for users

to test:
launch the web app
make a new user and upload a picture too
launch mongo on terminal
find the last user' s profile picture url, and open it on your browser, you should be able to see the picture
